graspologic/layouts/auto.py

```diff
@@ -260,7 +256,14 @@ def _node_positions_from(
     sizes = _compute_sizes(degree)
     covered_area = _covered_size(sizes)
     scaled_points = _scale_points(down_projection_2d, covered_area)
-    partitions = leiden(graph, random_seed=random_seed)
+    if isinstance(graph, nx.DiGraph):
```

Comment:
I think you'd be better off with `if graph.is_directed()` rather than doing it based on typing.  This way is probably fine but we may as well use the flag if it's there :)

Reply:
Sounds good, thank you!

Reply:
Just made the change!